backend/lib/endpoints/posts.js

```diff
@@ -367,11 +367,17 @@ async function routes(app) {
       const { userId } = req;
       const { postId } = req.params;
       const [findErr, post] = await app.to(Post.findById(postId));
+      const [orgErr, org] = await app.to(User.findById(userId));
+
```

Comment:
You might want to check if your eslint/prettier is working probably. Also, check if you have EXTEND_ESLINT=true
set true in your .env/client file.

Reply:
This space here it is fine. Bue yeah, It could be the local settings, or also the `pre-commit` hook that is again doing something messy when formatting.

Reply:
> Are you going to add the menu in the profile view as well in this PR?

yes trying that 👍

Reply:
yeah I have EXTEND_ESLINT=true in my `.env/client`

Reply:
@omarzoghayyer this one is backend so not related to `.env/client` file.

Also the pre-commit hook doesn't work currently with docker development #786.  So we have to remember to check ourselves until this issue is addressed.